src/hooks/auth0-provider.tsx

```diff
@@ -131,14 +145,14 @@ const Auth0Provider = ({domain, clientId, children}) => {
     }
   }, [client]);
 
-  const requireLocalAuthentication = useCallback(async (...options) => {
+  const requireLocalAuthentication = useCallback(async (...options: any) => {
```

Comment:
Would it make sense to provide a typed options parameter here instead of resorting to any?